waku/v2/node/wakuoptions.go

```diff
@@ -553,10 +551,7 @@ var DefaultLibP2POptions = []libp2p.Option{
 		libp2p.Transport(libp2pwebtransport.New),
 	),
 	libp2p.UserAgent(userAgent),
-	libp2p.ChainOptions(
-		libp2p.Muxer("/yamux/1.0.0", yamux.DefaultTransport),
-		libp2p.Muxer("/mplex/6.7.0", mplex.DefaultTransport),
-	),
+	libp2p.DefaultMuxers,
```

Comment:
Looks like mplex was removed by default in latest version. 
Maybe we should still enable mplex explicitly so that nodes which don't support yamux still work.

Reply:
yes

Reply:
@waku-org/js-waku-developers @waku-org/nwaku-developers shall we move to yamux?

Reply:
@fryorcraken https://github.com/waku-org/nwaku/issues/2331

Reply:
I think yes, it seems straightforward for `js-waku`

Reply:
https://github.com/waku-org/js-waku/issues/1771

Reply:
yes, sounds good.
yamux is recommended to be used over mplex by libp2p -- allows some stream handling enhancements over mplex & is fully supported by js-libp2p